src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +78,12 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> (p.getSex().equals(Person.Sex.MAN)
+                        && p.getAge() >= fromAge && p.getAge() <= maleToAge)
+                        || (p.getSex().equals(Person.Sex.WOMAN)
+                        && p.getAge() >= fromAge && p.getAge() <= femaleToAge))
```

Comment:
I would split it into two checks
```suggestion
                .filter(p -> p.getAge() >= fromAge)
                .filter(p -> p.getSex().equals(Person.Sex.MAN) && p.getAge() <= maleToAge
                        || p.getSex().equals(Person.Sex.WOMAN) && p.getAge() <= femaleToAge)
```